tokio-console/README.md

```diff
@@ -130,6 +188,18 @@ a large number of tasks, such as this private `tokio::sync::batch_semaphore::Sem
 
 ![resource details --- semaphore](https://raw.githubusercontent.com/tokio-rs/console/main/assets/tokio-console-0.1.8/resource_details_semaphore.png)
 
+The resource details view includes a table of async ops belonging to the resource.
+
+* `ID` - The ID of the async op, this is a display ID similar to the one for resources
```

Comment:
```suggestion
* `ID` - The ID of the async op. This is a display ID similar to those recorded for resources. 
```

Reply:
I'd prefer to use multiple sentences, personally. I'm fine with adding a full stop to every bullet point if that lets us break some of these into two sentences instead of using a comma.

Reply:
Fair enough. I've done that and broken up the sentences as you suggested.